src/orion/core/evc/conflicts.py

```diff
@@ -1608,7 +1608,7 @@ def _check_for_children(self):
             # If we made it this far, new_name is actually the name of the parent.
             parent = self.conflict.old_config
 
-            query = {'refers.parent_id': parent['_id']}
+            query = {'name': parent['name'], 'refers.parent_id': parent['_id']}
```

Comment:
This would only fetch children with same name, thus not all children. I understand this is what we are looking for here, but then the methods name become very confusing. `_check_for_greater_versions` maybe?

Reply:
Sounds good.